chromium_src/chrome/common/chrome_constants.cc

```diff
@@ -201,6 +206,13 @@ const int kLowestRendererOomScore = 300;
 const int kHighestRendererOomScore = 1000;
 #endif
 
+#if defined(OS_CHROMEOS)
+const char kProfileDirPrefix[] = "u-";
```

Comment:
do we ever hit this code-path?

Reply:
Doubt it, i just copied things from  the original so it's easier to diff them in the future.